homework3/semi-uber/UI/src/app/registration-page/registration-page.component.html

```diff
@@ -90,9 +90,11 @@
       />
     </label>
 
-    <button class="registration__button button" (click)="getRegForm()">
+    <button class="registration__button button" (click)="registrateUser()">
       Submit
     </button>
   </div>
 </form>
-<span class="registration__error" *ngIf="formField">All fields must be filled</span>
+<span class="registration__error" *ngIf="errorFillFields"
```

Comment:
I've already written you about the validation. As for me it is better to show error messages under every invalid input with the corresponding error. For example if user started typing email, but it is not valid and user became typing in the other input, you should display `Invalid email`. So every input that has some special validation rules should display the corresponding error for every concrete validation rule, that is not valid

Reply:
Notice it for future